aerospike/metadata.csv

```diff
@@ -163,6 +163,15 @@ aerospike.migrate_msgs_recv,gauge,,,,The number of migrate messages received. De
 aerospike.migrate_msgs_sent,gauge,,,,The number of migrate messages sent. Deprecated since 3.8.3,0,aerospike,,
```

Comment:
For non-Openmetrics metrics, can you add `[Legacy]` in the description? And can you add the `[Legacy and OpenMetrics V2]` in the descriptions of checks that can be both? See [the Istio metadata.csv for example](https://github.com/DataDog/integrations-core/blob/master/istio/metadata.csv)